icons/goal.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 13V2l8 4-8 4" />
+  <path d="M20.6 10.2A9 9 0 1 1 8 4.94" />
+  <path d="M8 10a5 5 0 1 0 8.9 2.02" />
+</svg>
```

Comment:
```suggestion
</svg>

```

Reply:
It is customary to add a new line character at the ends of files.

Reply:
Understood! Thank you